docs/pages/includes/database-access/start-auth-proxy.mdx

```diff
@@ -1,27 +1,41 @@
+Teleport requires a valid TLS certificate to operate and can fetch one automatically
+using Let's Encrypt's [ACME](https://letsencrypt.org/how-it-works/) protocol. 
```

Comment:
Should this be under self-hosted section? I don't think this is relevant to Cloud users.